Build failure

```
vi.c(750): error C2220: warning treated as error - no 'object' file generated [C:\mg\angrylion-rdp-plus\core\core.vcxproj]
vi.c(750): warning C4098: 'vi_update': 'void' function returning a value [C:\mg\angrylion-rdp-plus\core\core.vcxproj]
```

Mupen64plus build in Visual Studio

--- a/core/vi.c
+++ b/core/vi.c
@@ -746,9 +746,9 @@ void vi_update(void)
     hres =  h_end - h_start;
     vres = (v_end - v_start) >> 1; // vertical is measured in half-lines
     
-    if (hres <= 0 || vres <= 0) {
-        return 0;
-    }
+    // skip invalid/unsupported frame sizes
+    if (hres <= 0 || vres <= 0)
+       return;
 
     x_add = *vi_reg_ptr[VI_X_SCALE] & 0xfff;
     x_start = (*vi_reg_ptr[VI_X_SCALE] >> 16) & 0xfff;

